issues_rss.php

```diff
@@ -172,17 +172,18 @@
 }
 $t_show_sticky = null;
 
+# Override current user
+current_user_set( $t_user_id );
```

Comment:
I usually use `auth_attempt_script_login()` for impersonating a user for running some logic.  It has more logic than what `current_user_set()`.

Reply:
As i see: `rss_login()'  which is used previously in the code, is already calling `auth_attempt_script_login()`
I'm not sure what's is best here. Since this may imply that a user (named, or anonymous) has been logged in by those calls, may this `current_user_set()` be simply removed?